generators/client/templates/angular/src/main/webapp/app/account/_account.module.ts

```diff
@@ -50,7 +50,7 @@ import {
 @NgModule({
     imports: [
         <%=angularXAppName%>SharedModule,
-        RouterModule.forRoot(accountState, { useHash: true })
```

Comment:
Is usehash true by default?

Reply:
Else it will fail as we use hashbang urls. As far as i remember its not true by default so please check

Reply:
We have a global `forRoot`, I did some tests the child inherit the `root` behavior (with hashbang).
Either the `generator-jhipster` and the application tests are green

Reply:
Ok thanks